apis/beacon/blocks/blocks.yaml

```diff
@@ -24,9 +25,9 @@ post:
     "200":
       description: "The block was validated successfully and has been broadcast. It has also been integrated into the beacon node's database."
     "202":
-      description: "The block failed validation, but was successfully broadcast anyway. It was not integrated into the beacon node's database."
+      description: "The block failed full validation, but was successfully broadcast anyway. It was not integrated into the beacon node's database."
```

Comment:
Maybe 
`The block could not be integrated into the beacon node's database as it failed validation, but was successfully broadcast.`